Generic_CCVA.Rmd

```diff
@@ -194,13 +202,17 @@ max_overlap_cent <- centroid_sel[centroid_sel$n == max(centroid_sel$n)][[3]]
 # Import table of selected futures for site. Selections were made using the "select_GCMs" 
 # function. 
 select_cfs <- data.table::fread('data/parkwide_GCMs.csv') %>% 
+              mutate_at("centroid", str_replace, "_future", "") %>%
               dplyr::filter(park %in% {{park}}) %>%
               # Uncomment either:
               # a. if using min distance to supply watershed 
               dplyr::filter(centroid == min_dist_cent)
               # or b.if using max overlap with supply watershed
               #dplyr::filter(centroid == max_overlap_cent)
 
+# replacement for now until selection is fixed
+select_cfs[select_cfs$GCM=="MIROC-ESM.rcp85",]$GCM <- "MIROC-ESM-CHEM.rcp85"
+
 # ****KEC: update function to check if files expist before downloading.
 get_centroid_climate_data(park = park, save = TRUE)
```

Comment:
Is the purpose of this to just write these files to the data folder?

Reply:
Seems like it - so if data is already in there, we skip this step